conda_build/cli/main_index.py

```diff
@@ -96,6 +98,11 @@ def parse_args(args):
 def execute(args):
     _, args = parse_args(args)
 
+    if args.file:
+        warnings.warn("--file is no longer implemented.")
+    if args.hotfix_source_repo:
+        warnings.warn("--hotfix-source-repo is no longer implemented.")
+
```

Comment:
These aren't available in the new conda-index, right? Are there equivalent functionalities? Should people be warned with a `DeprecationWarning` before we do remove these?

Reply:
`--hotfix` was removed before I started working on conda-build and `--file` isn't implemented in new conda-index. The replacement is that new conda-index is fast.